__tests__/app.test.js

```diff
@@ -134,4 +134,36 @@ describe("GET /api/articles", () => {
       expect(msg).toBe('bad request')
     })
   })
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test('200: returns an array of all comments related to a particular article, sorted by date', () => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then(({body}) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        expect(comments).toBeSortedBy('created_at')
+        comments.forEach((obj) => {
+            expect(obj).toMatchObject({
+                author: expect.any(String),
+                article_id: expect.any(Number),
```

Comment:
You are not expecting any article id. You are specifically requesting comments from article 1. If this were article 12 this would still pass.